examples/kotlin/pom.xml

```diff
@@ -184,6 +184,7 @@
                         <disableDefaultRuleSets>true</disableDefaultRuleSets>
                         <classPath>${generated.classpath}</classPath>
                         <jvmTarget>${java.version}</jvmTarget>
+                        <jdkHome>${java.home}</jdkHome>
```

Comment:
Super bizarre, you'd think it would be able to figure this out itself...

Reply:
i know 😢  thankfully i came across [this issue](https://github.com/detekt/detekt/pull/5269), which pointed me to this solution